src/middlewared/middlewared/plugins/filesystem.py

```diff
@@ -302,6 +297,132 @@ def acl_is_trivial(self, path):
         a = acl.ACL(file=path)
         return a.is_trivial
 
+    @accepts(
+        Dict(
+            'filesystem_ownership',
+            Str('path', required=True),
+            Int('uid', null=True, default=None),
+            Int('gid', null=True, default=None),
+            Dict(
+                'options',
+                Bool('recursive', default=False),
+                Bool('traverse', default=False)
+            )
+        )
+    )
+    def chown(self, data):
+        """
+        Change owner or group of file at `path`.
+
+        `uid` and `gid` specify new owner of the file. If either
+        key is absent or None, then existing value on the file is not
+        changed.
+
+        `recursive` performs action recursively, but does
+        not traverse filesystem mount points.
+
+        If `traverse` and `recursive` are specified, then the chown
+        operation will traverse filesystem mount points.
+        """
+        self.logger.debug(f'chown: {data}')
```

Comment:
Shouldn't this be `trace`?

Reply:
Actually, that shouldn't be there at all. Will remove.